ports/qtbase/vcpkg.json

```
@@ -336,6 +343,10 @@
         }
       ]
     },
+    "securetransport": {
+      "description": "Enabled Secure Transport",
```

```suggestion
      "description": "Enable Secure Transport",
```
Sorry for the nitpick, just to be consistent with the other features

---

@m-kuhn , good point, done.